.github/renovate.json

```diff
@@ -51,6 +51,11 @@
       "versioning": "loose",
       "ignoreUnstable": false
     },
+    {
+      "matchPackageNames": ["io_opentelemetry_cpp", "open-telemetry/opentelemetry-cpp"],
+      "groupName": "OpenTelemetry",
+      "versioning": "loose"
+    },
     {
```

Comment:
Optional: consider adding groups for `benchmark` and `googletest`.

Reply:
Done. Added `nlohmann/json` too, because we get renovate PRs for it.

I noticed that we do not get PRs for `googletest`. The [latest release](https://github.com/google/googletest/releases/tag/release-1.12.1) was 1.12.1. I am not sure how it is escaping our regex.... https://github.com/googleapis/google-cloud-cpp/blob/356b90243b440aff62ff0a326d097fbbc143ea08/.github/renovate.json#L14

Reply:
Possible needs the "loose" versioning too. This log may have a clue:

https://app.renovatebot.com/dashboard#github/googleapis/google-cloud-cpp